lib/src/accessibility_tools.dart

```diff
@@ -27,7 +28,9 @@ class AccessibilityTools extends StatefulWidget {
     required this.child,
     this.minimumTapAreas = MinimumTapAreas.material,
     this.checkSemanticLabels = true,
+    this.checkMissingInputLabels = true,
     this.checkFontOverflows = false,
+    this.printResolutionGuidance = true,
```

Comment:
Hmm... not sure this is that useful as an option. Maybe leave it out and see if anyone requests it?

Reply:
It's gone. I've also adjusted how guidance is printed so the order is now:
* Issue name
* Error causing widget
* Resolution guidance

Previously resolution was printed before error causing widget, I think it makes more sense this way.

Also found a test that was testing semantic label instead of min tap area, fixed [here](https://github.com/rebelappstudio/accessibility_tools/blob/52a9535d52ffbf5fa06953f7c9313cbec39d2f01/test/minimum_tap_area_checker_test.dart#L80).